test/test_helper.exs

```diff
@@ -1 +1,10 @@
 ExUnit.start()
+
+# cleanup any streams left over by previous test runs
```

Comment:
Also added this test helper which cleans up streams at the beginning of the test suite incase we had leftovers from a previous run